cmd/thor/flags.go

```diff
@@ -183,6 +183,12 @@ var (
 		Usage: "comma separated list of allowed API tracers(none,all,call,prestate etc.)",
 	}
 
+	minMaxPriorityFeePerGasFlag = cli.Uint64Flag{
+		Name:  "minimum-max-priority-fee-per-gas",
+		Value: 0,
+		Usage: "minimum max priority fee per gas for all transactions",
```

Comment:
```suggestion
		Usage: "block proposer only flag, to set a minimum priority fee per gas for transactions to be included into block",
```

This description doesn't express that the value is checked via `>`, maybe we should set this flag's default value to 1?

Reply:
I think 0 is the correct value for the following reasons:

1. If 1, we are essentially making the flag required since even though we do not set anything we will be requesting a `maxPriorityFeePerGas` >= 1 (so we wont be allowing 0s in practice)
2. If 1 is the minimum value, rather than a flag the value should be set at the attribute level
3. The description is accurate because the value set there is effectively the minimum, so >= 0. At `flow.go` we have the following:

```
	maxPriorityFeeTooLow := t.MaxPriorityFeePerGas() == nil || t.MaxPriorityFeePerGas().Cmp(f.minTxPriorityFee) < 0
	if flagIsSet && maxPriorityFeeTooLow {
		return badTxError{"max priority fee per gas too low"}
	}
```

So `t.MaxPriorityFeePerGas().Cmp(f.minTxPriorityFee) >= 0` (the opposite) would allow transactions to be adopted.

Reply:
I see your point, sorry for the confusing. I am considering the minimum `effectivePriorityFeePerGas` should be 1 otherwise validators are doing works for free, would you agree?